pkg/rancher/rancher.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/rancher/rancher/pkg/controllers/dashboard/plugin"
 	"github.com/rancher/rancher/pkg/controllers/dashboardapi"
 	managementauth "github.com/rancher/rancher/pkg/controllers/management/auth"
+	"github.com/rancher/rancher/pkg/controllers/management/kontainerdrivermetadata"
```

Comment:
It might be a good idea to move this, since all of the management controllers are typically started here: `pkg/controllers/management/controller.go`. Moving it to another package might make more sense.

Reply:
@jakefhyde are you suggesting to move `kontainerdrivermetadata` pkg out of `controllers/management` directory?

Reply:
moved to a separate directory under `pkg`